.github/workflows/shadow-repo.yml

```diff
@@ -8,7 +8,7 @@ on:
     types: [opened]
```

Comment:
@jgerigmeyer I think we only need it here because the workflow is fired by a bunch of event types, and for each we do something different. `test.yml` only listens for a couple and they don't happen at the same time.